tests/Finder/SourceFilesFinderTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Copyright © 2017 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Tests\Finder;
+
+use Infection\Finder\SourceFilesFinder;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * @author Lander Vanderstraeten <lander_vanderstraeten@hotmail.com>
+ */
+class SourceFilesFinderTest extends TestCase
+{
+    public function test_it_lists_all_files_without_a_filter()
+    {
+        $sourceFilesFinder = new SourceFilesFinder(['tests/Files/Finder'], ['vendor']);
+
+        $files = $sourceFilesFinder->getSourceFiles();
+
+        $this->assertInstanceOf(Finder::class, $files);
+        $this->assertEquals(2, $files->count());
```

Comment:
`$this->assertSame()`, it's safer in general and avoid surprises with unexpected casting  (same elsewhere)

Reply:
Should we check here what are those two files? (same elsewhere)